packages/gatsby-theme-docs/src/components/link.js

```diff
@@ -96,10 +96,10 @@ export const ExternalSiteLink = (props) => (
  */
 const PureLink = (extendedProps) => {
   const siteData = useSiteData();
-  const { location, noUnderline, ...props } = extendedProps;
+  const { location, noUnderline, ariaLabel, ...props } = extendedProps;
```

Comment:
There should be no need to pass an `aria-label`. You can access the link by the text, for example `findByText('This is a link')`

Reply:
> There should be no need to pass an `aria-label`. You can access the link by the text, for example `findByText('This is a link')`

A link text can exist in the main navigation as well as a page, doing this for me ensures I am getting the right text.

Do you still think an aria-label is not needed?

Reply:
If there are multiple links in the page with the same text, you can narrow down the selection by selecting links within certain containers. There are different ways to do that with Cypress.

Still, an `aria-label` is still not necessary, as you have probably the same issue selecting the right link.